lib/std/os.zig

```diff
@@ -268,9 +268,9 @@ pub fn kill(pid: pid_t, sig: u8) KillError!void {
 }
 
 /// Exits the program cleanly with the specified status code.
-pub fn exit(status: u8) noreturn {
+pub fn exit(status: u32) noreturn {
     if (builtin.link_libc) {
-        system.exit(status);
+        system.exit(@intCast(c_int, status));
```

Comment:
This would cause safety-checked illegal behavior if the user chooses a status code outside the range of c_int. Are you sure this is the right decision? Can you do some research into the valid range of exit code values on various systems for some context? Are we sure that u32 is the correct value for child process exit code?

Reply:
See https://unix.stackexchange.com/a/418802/19884

Reply:
Nice find @daurnimator. That's the way to do this thing. Closing; please re-open with this alternative implementation strategy.